clang/test/CXX/drs/cwg13xx.cpp

```diff
@@ -451,6 +463,25 @@ static_assert(!__is_nothrow_constructible(D4, int), "");
 #endif
 } // namespace cwg1350
 
+namespace cwg1352 { // cwg1352: 3.0
+struct A {
+#if __cplusplus >= 201103L
+  int a = sizeof(A);
```

Comment:
I think it might be worth it to see that this fails for a `static member` since the DR specifically says `non-static` so we should cover both.

Reply:
It also mentions in the body of member function

Reply:
It's not going to fail for static member initializers because of (I suspect) subsequent changes that were made to the wording: http://eel.is/c++draft/class.mem.general#8.5

Reply:
> It also mentions in the body of member function

Yeah, that is worth adding. I'll prepare a PR.